Tools/clinic/clinic.py

```diff
@@ -5409,29 +5496,31 @@ def parse_converter(
                     "Annotations must be either a name, a function call, or a string."
                 )
 
-    def parse_deprecated_positional(self, thenceforth: str) -> None:
+    def parse_version(self, thenceforth: str) -> VersionTuple:
```

Comment:
Could you add a one-line docstring here explaining what this function does, similar to the other `parse_foo` methods on this class?